lib/passport-configurator.js

```diff
@@ -35,6 +35,8 @@ function PassportConfigurator(app) {
  * @property {Model} [userModel] The user model class
  * @property {Model} [userCredentialModel] The user credential model class
  * @property {Model} [userIdentityModel] The user identity model class
+ * @property {Array} [additionalUserModels] Additional model classes belonging to user model
+ * @property {Array} [additionalModels] Additional model classes
```

Comment:
The type should be `Object` instead of `Array`, right?

Reply:
Sorry, you are correct, I will fix that now

Reply:
@raymondfeng This has been resolved.